scheduler/remote_server.py

```diff
@@ -0,0 +1,487 @@
+import inotify.adapters
+import time
+import scd_utils
+import email_utils
+import os
+import datetime
+import argparse
+import collections
+import math
+import copy
+import random
+import subprocess as sp
+import pickle as pkl
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+
+import remote_server_options as rso
+
+def format_to_atq(dt, experiment, first_event_flag=False):
+    """Turns an experiment line from the scd into a formatted atq command.
+
+    Args:
+        dt (datetime): Datetime of the experiment
+        experiment (str): The experiment to run
+        first_event_flag (bool, optional): Flag to signal whether the experiment is the first to
+        run
+
+    Returns:
+        str: Formatted atq str.
+    """
+    if first_event_flag:
+        cmd_str = "echo '{borealis_path}/start_radar.sh {experiment}' | at now"
+        cmd_str = cmd_str.format(borealis_path=os.environ['BOREALISPATH'],experiment=experiment)
+        cmd_str = dt.strftime(cmd_str)
+    else:
+        cmd_str = "echo '{borealis_path}/start_radar.sh {experiment}' | at -t %Y%m%d%H%M"
+        cmd_str = cmd_str.format(borealis_path=os.environ['BOREALISPATH'],experiment=experiment)
+        cmd_str = dt.strftime(cmd_str)
+
+    return cmd_str
+
+def get_next_month(date):
```

Comment:
is this the only duplicate function? otherwise could be a staticmethod in scdutils perhaps (used in local_scd_server but that has scdutils import)

Reply:
Its not exactly the same function, but ill rename it